extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -82,16 +82,23 @@
     private Logger logger = LoggerFactory.getLogger(HueLightHandler.class);
 
     // Flag to indicate whether the bulb is of type Osram par16 50 TW or not
-    private boolean isOsramPar16 = false;
+    private boolean isOsramPar16;
 
-    private boolean propertiesInitializedSuccessfully = false;
+    private boolean propertiesInitializedSuccessfully;
 
     private HueBridgeHandler bridgeHandler;
 
     ScheduledFuture<?> scheduledFuture;
 
     public HueLightHandler(Thing hueLight) {
         super(hueLight);
+        lightId = null;
+        lastSentColorTemp = null;
+        lastSentBrightness = null;
+        isOsramPar16 = false;
+        propertiesInitializedSuccessfully = false;
+        bridgeHandler = null;
+        scheduledFuture = null;
```

Comment:
As stated by @9037568 the chapter 4.12.5 contains a list of the default values:
http://docs.oracle.com/javase/specs/jls/se8/html/jls-4.html#jls-4.12.5